### ISSUE TITLE
[sozo] Typescript bindings (bytes31 / ContractAddress)

### ISSUE THREAD

**Issue by notV4l** [dojoengine/dojo#1626]
**Is your feature request related to a problem? Please describe.**
I had this 2 errors trying to use `sozo build --typescript` 

```
thread 'main' panicked at crates/dojo-bindgen/src/plugins/typescript/mod.rs:58:40:
Token not found: bytes31
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

```
thread 'main' panicked at crates/dojo-bindgen/src/plugins/typescript/mod.rs:271:50:
called `Result::unwrap()` on an `Err` value: ConversionFailed("Can't convert token into composite, got CoreBasic(CoreBasic { type_path: \"core::starknet::contract_address::ContractAddress\" })")
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Additional context**
v0.6.0-alpha-3

**Comment by Larkooo** [comment]
bytes31 support hasn't been added. what project are you using? contractaddress bug should be fixed tho, if you use the latest branch. could you link me to your project so i can fix the relevants token types?

**Comment by Larkooo** [comment]
The second issue should have been resolved. It has to do with the same issue with unity

**Comment by notV4l** [comment]
> @Larkooo I will double check into cainome how this type could be handled.
> 
> For the `ContractAddress` in the second error, would aslo be interested into the repo @notV4l. 👍

it was on origami afair

**Comment by glihm** [comment]
Should have been resolved by #1847.
